poc_config/openmrs/apps/common/formLayout.json

```diff
@@ -244,11 +245,11 @@
                     },
                     {
                         "name": "Frequencia Cardiaca",
-                        "id": "a32767e4-3b41-411e-ad5d-a989cedaab40"
+                        "id": "95c5a7ad-c43e-4a0f-8886-ee3cba1b863d"
```

Comment:
I don't think this PR was ready to be merged. For example, we need some clarity on these UUID changes. As far as I know, @valvijo asked @Benilde to create these concepts manually in her database (because we had that issue with sharing concepts via MDS). So, I think we should actually have kept the existing values, since I think the plan might be to share those via MDS.

@valvijo and/or @Benilde, can you comment on this?